docs/specifications/tests/Basic-TP/basic04.md

```diff
@@ -21,7 +21,7 @@
 
 Many test cases will query the name servers in the delegation or the
 name servers appointed by the NS records in the zone for the NS or SOA
-record or both. Reporting problem is crucial, but instead of letting
+record, or both. Reporting problem is crucial, but instead of letting
```

Comment:
Reporting problem -> Reporting problems

Reply:
I do not like "problems is" so I rephrased it.